sdk/core/src/shared/SdkErrorFromCode.ts

```diff
@@ -146,6 +146,11 @@ export default class SdkErrorFromCode extends SdkError {
         message: 'did method: {{didMethod}} not supported. Supported methods: {{supportedDidMethods}}',
         httpStatusCode: 409,
       },
+      'COR-26': {
```

Comment:
But it is not thrown anywhere?

Reply:
I was added this initially, but don't want overwrite existing errors. So will remove it.